cgd_tx_eff/src/edu/ohsu/compbio/txeff/tx_eff_hgvs.py

```diff
@@ -610,7 +591,7 @@ def _merge_into(self, transcript_key: str, new_transcript: VariantTranscript, hg
     
         # c-dot
         ## Use HGVS's c. because Annovar's is not always correct. Non-coding transcripts don't have a c. 
-        assert hgvs_transcript.hgvs_c_dot != None or hgvs_transcript.refseq_transcript.startswith('NR_'), "The HGVS c. value is not supposed to be empty"
+        assert hgvs_transcript.hgvs_c_dot or hgvs_transcript.refseq_transcript.startswith('NR_'), "The HGVS c. value is not supposed to be empty"
```

Comment:
If we're excluding NR_ transcripts, we should be able to remove the four NR_ assertions from this method, right?